torch/distributions/f.py

```diff
@@ -0,0 +1,76 @@
+from numbers import Number
+import torch
+import math
+from torch.distributions import constraints
+from torch.distributions.distribution import Distribution
+from torch.distributions import Beta, Chi2
+from torch.distributions.utils import broadcast_all
+
+
+class F(Distribution):

```

Comment:
naming: `F` is a little dangerous for a class name, e.g. it is common in PyTorch to
```py
import torch.nn.functional as F
```
Consider lengthening the name to `FDistribution` or `FisherSnedecor` or `SnedecorF`.